content/en/docs/reference/glossary/primary-cluster.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Primary Cluster
+test: n/a
+---
+
+A primary cluster is a cluster with a control plane, which observes all
+clusters in the mesh for endpoints and facilitates cross-cluster load balancing
+between clusters.
```

Comment:
Things we should mention:
- Multiple primary clusters can be connected together in a single mesh (point to glossary entry for mesh)
- Primary clusters can act as the control plane for remote clusters (point to glossary entry for remote cluster).

I think the first one can replace the note about observing endpoints and facilitating cross-cluster LB, which seems a bit too much implementation detail of how they are connected, and not really part of the definition of a primary cluster.